db/migrate/20230303192614_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :likes, :author, null: false, foreign_key: { to_table: 'users' }
+    end
+  end
```

Comment:
- [OPTIONAL] Hello @187jjay187, everything looks fine However as requested by the previous reviouwer, This error message indicates that there was an error while running a database migration, which caused the migration - process to be aborted.
Here are some steps you can take to diagnose and fix the issue:
  
1. Check the database schema: Verify that the database schema matches the expected schema defined in your migration files.
2. Roll back the migration: If you're still having trouble, you may need to roll back the migration and try again. You can do this by running the rails db:rollback command.

Reply:
@BenMukebo thank you so much for the help and directions I really do appreciate it and will definitely look into it before trying to move on